### ISSUE TITLE
Unwanted Changes when hitting escape key

### ISSUE THREAD

**Issue by islamadel** [subethaedit/SubEthaEdit#178]
When the cursor is at any text and I hit the escape key, it starts a modifiying toggle (autocompletion) to my text with unwanted changes!
Just hitting the escape key once already applies changes!
Sometimes the cursor is somewhere above or below the current view, where I don't see it. so the changes take place without knowing.
I saved my files multiple times with unwanted changes, because of the escape key!!
Please tell me how to disable this toggle forever.

**Comment by monkeydom** [comment]
Ah, ic. I do think this is not new behavior, but I take away that any action that is not tab or escape should remove the text again if there is ongoing autocomplete. So you don't have an accidential save of garbage. 

Currently only another escape press or entering other text does remove the current autocomplete suggestion.

**Comment by islamadel** [comment]
yeah another escape would remove it, but in case the cursor is not within the view field and you keep hitting escape, you would not notice if the text has changed or not. you would have to keep scrolling searching for the cursor and still hit escape at least twice to see what change has been done
